src/connection/base.py

```diff
@@ -0,0 +1,34 @@
+from abc import ABC, abstractmethod
+
+import aiohttp
+
+from src.api.api import ApiClient
+from src.http.http import Response, Request, ResponseException
```

Comment:
from src.http.http import cośtam
    from src.api.api import cośtam

Ja nie wiem, może tak się robi. Ale po co tworzyć folder, który ma w sobie tylko jeden plik z kodem źródłowym, którego nazwa jest tożsama z nazwą folderu?

Osobiście bym to "spłaszczył", zrezygnował z folderów, powsadzał wszystkie pliki do folderu /src i niech tam leżą, póki nie nadejdzie konieczność potworzenia podfolderów w /src.

Nie będę się jednak upierał

Reply:
Chociaż z drugiej strony jeśli damy te modele to one winne leżeć we własnym folderze

tak sobie teraz przypomniałem

to może tak:

folder: /src/models/ a w nim modele
folder: /src/connection a w nim oauth hmac connectionbase itp
plik: /src/api.py
plik: /src/tools.py

? 

Dobry pomysł czy nie?

Reply:
moze byc